MVCDemos/Views/Form/GroupedFields.cshtml

```diff
@@ -25,12 +27,14 @@
 
             items.AddGroup()
                 .Caption("Personal Data")
+                .CaptionTemplate(new JS("function(data) { return groupCaptionTemplate('accountbox', data); }"))
```

Comment:
"Let's replace the 'accountbox' icon with a 'card' icon. Please refer to the latest sources for implementing this change in jQuery, React, Angular, and Vue. Begin with the jQuery: approachhttps://github.com/DevExpress/DevExtreme/blob/c52f4d8587f9a80fee5cecb180157acc33f98589/apps/demos/Demos/Form/GroupedFields/jQuery/index.js#L24